### ISSUE TITLE
Whitelist dock.so

### ISSUE THREAD

**Issue by amkracov** [MetaMask/eth-phishing-detect#5017]
Hi there - 

I'm the owner of the domain - dock.so - and it recently came to my attention that we're listed flagged as spam because of this list. 

We're new owners of this domain (purchased Dec 2020), so I was surprised to find that we're listed in this database, especially since we have nothing to do with crypto. My guess is the previous owners of the domain were bad actors. 

Can you please remove this domain from your database? What can we do to remedy the situation? 

TLDR: Please whitelist dock.so

Thanks, 

Alex
